src/lib/Scenes/Show/Screens/Detail.tsx

```diff
@@ -134,19 +134,6 @@ export class Detail extends React.Component<Props, State> {
     return null
   }
 
-  handleAnimationFrame = animatedValue => {
-    /**
-     * If children change their size on animation (e.g. HoursCollapsible), we need a sentinel value
-     * in state in order to trigger a re-render, as FlatList statically sizes child cells.
-     */
-    this.setState({
```

Comment:
@orta @kierangillen This is the root cause of the animation jerkiness. This callback was being called at every frame (~60fps) and calling `setState` which would potentially trigger a render pass (only potentially because it's an async call). After removing this, the animation performed

<img src="https://user-images.githubusercontent.com/296775/53392558-c0c40f80-3967-11e9-9f77-9ee5699542e9.gif" width="400" />

Reply:
heeyyyyyyy